specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2022-05-01/machineLearningServices.json

```diff
@@ -2204,6 +2204,10 @@
           "description": "The URI associated with this workspace that machine learning flow must point at to set up tracking.",
           "type": "string",
           "readOnly": true
+        },
+        "v1LegacyMode": {
+          "description": "If the workspace is in V1 legacy mode",
```

Comment:
Also in the description can you add something that helps understand that this flag is in combination with private link

Reply:
Actually, for client-side perspective this flag is not related to PL and they can turn it regardless of other flags, I added the MFE expected behaviour here.